app/presenters/order_detail_notice_presenter.rb

```diff
@@ -0,0 +1,85 @@
+class OrderDetailNoticePresenter < DelegateClass(OrderDetail)
+
```

Comment:
None that are super compelling. Slight performance improvement, and better introspection, `OrderDetailNoticePresenter.instance_methods` includes the OrderDetail methods when it's `DelegateClass`, but doesn't know about them if it's `SimpleDelegator`.

I'd be happy to switch if you think there's a good reason to prefer `SimpleDelegator`.